CONTRIBUTING.md

```diff
@@ -138,8 +139,8 @@ New features:
 ### Devops
 New features:
 
-- If there are any new configuration properties, do the follow the same
-  structure as existing configuration style?
+- Does the configuration style follow the config guidelines? That is compile
+  (Maven) config goes in `pom.xml`, runtime (Spring) goes in `portal.properties`.
```

Comment:
I’m not sure if the line about `pom.xml` adds much here, as there’s multiple files with that name in the project, and I expect that Maven has its own documentation on where to configure the build process.

I do think it would be useful to add that defaults for anything in `src/main/resources/portal.properties` (which is really an installation-specific configuration file and therefore not under version control) go in `core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java`.